### PR TITLE
Fix shape of logits in regression head

### LINKED ISSUE #229: Using a target size ... that is different to the input size
Please what does this mean ?

Train epoch 1/20 (Cur. train loss: 218.7825):  61%|██████▏   | 62/101 [01:40<02:25,  3.72s/it]/home/ec2-user/anaconda3/envs/python3/lib/python3.6/site-packages/torch/nn/modules/loss.py:431: UserWarning: **Using a target size (torch.Size([32, 1])) that is different to the input size (torch.Size([32]))**. This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.

### PR DESCRIPTION
**Pull request by tholor**
Fixes #229

### CODE REVIEW

**Comment by zolekode** [comment]
Yes the error is gone. Awesome
